test/SDIR/Dialect/memlet/alloc_param_too_few.mlir

```diff
@@ -4,13 +4,16 @@
 // CHECK: module
 // CHECK: sdir.sdfg
 sdir.sdfg{entry=@state_0} @sdfg_0 {
-    // CHECK-NEXT: [[NAMEn:%[a-zA-Z0-9_]*]]
-    %n = arith.constant 5 : index
-    // CHECK-NEXT: {{%[a-zA-Z0-9_]*}} = sdir.alloc([[NAMEn]], [[NAMEm]])
-    // CHECK-SAME: !sdir.array<?x?xi32>
-    %a = sdir.alloc(%n) : !sdir.array<?x?xi32>
-
     sdir.state @state_0{
-
+        // CHECK: sdir.tasklet @five
+        sdir.tasklet @five() -> index{
```

Comment:
would be nice to add a `TODO(later)` somewhere in the code to make a pass that adds those tasklets

Reply:
There is no need. Other dialects (apart from a funcOp) are disallowed in nodes (except tasklets). Adding a `arith.constant` in a node wouldn't even parse, which means no pass can be applied.